nestjs/src/courses/course-schedule/course-schedule.service.ts

```diff
@@ -360,7 +360,7 @@ export class CourseScheduleService {
   private getEventStatus(courseEvent: CourseEvent) {
     const startTime = (courseEvent.dateTime as Date).getTime();
     const endTime = Number(courseEvent.endTime) ?? startTime + (courseEvent.duration ?? 60) * 1000 * 60;
-    if (endTime < Date.now()) {
+    if (endTime && endTime < Date.now()) {
```

Comment:
just curious how `endTime` could be `falsy`?

Reply:
nevermind, find the way)

Reply:
@I-vasilich-I 
1) At this line it can't be due to expression above (please take a look at my comment)
2) Afair previously `endTime` was not mandatory field, so we have events where it's `null`

Reply:
> @I-vasilich-I
> 
> 1. At this line it can't be due to expression above (please take a look at my comment)
> 2. Afair previously `endTime` was not mandatory field, so we have events where it's `null`

at this line it could be `falsy` when `endTime` is `null` or `undefined`, which could be fixed by you proposal